lib/erl_interface/include/ei.h

```diff
@@ -329,7 +329,7 @@ typedef struct {
     int (*socket)(void **ctx, void *setup_ctx);
     int	(*close)(void *ctx);
     int (*listen)(void *ctx, void *addr, int *len, int backlog);
-    int (*accept)(void **ctx, void *addr, int *len, unsigned tmo);
```

Comment:
Just noticed this change.... this is a backward incompatible change, which means that there needs to be a very good reason to change it, and I'm not sure AIX support qualifies.

Not sure how to work around it except only renaming the field on AIX, though that is not a very pretty solution.

Reply:
There is the possibility of `#undefine accept` but then that gets crumbly once you try using system accept calls.

I could try renaming it back and do an `#ifdef` approach though.

Reply:
thanks for the super quick response. I'll put this in test over the weekend and then merge.